app/controllers/concerns/auth.rb

```diff
@@ -18,18 +18,6 @@ module Auth
     end
   end
 
-  def current_user
-    @current_user ||= if session.dig(:userinfo)
-      User.active.includes(:organisation).find_by!(identifier: signed_in_user_identifier) do |user|
-        user.name = session.dig(:userinfo, "info", "name")
-        user.email = session.dig(:userinfo, "info", "email")
-      end
-    end
-  rescue ActiveRecord::RecordNotFound
-    repudiate!
-    raise UserNotAuthorised
```

Comment:
Because we've removed the only thing that could `raise UserNotAuthorised` and the only usage of it is on L10 of this file, we should remove the class and also the branch of the `if` on L11 that does not respond to `:policy`